### DISCUSSION TITLE
Difference between Github.io and local browser?

### DISCUSSION THREAD

**Discussion by deonprovost** [community/community#21935] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>We have a working project at:</p>
<p><a href="https://gduer.github.io/CityGuesser/#" rel="nofollow noopener">https://gduer.github.io/CityGuesser/#</a></p>
<p>After forking the files located at: </p>
<p><a href="https://github.com/primetime5502/CityGuesser" rel="nofollow noopener">https://github.com/primetime5502/CityGuesser</a></p>
<p>The index.html file does not render. How is this possible? The <a href="http://github.io" rel="nofollow noopener">github.io</a> version in all browsers, but not when calling the forked file from local browser. We’re at a hackathon, please help!!!</p>

**Comment by darrowby385** [comment, marked as answer]
<p>Hi, <a class="mention" href="/primetime5502">@primetime5502</a>!</p>
<p>It sounds like you’ve created a fork of a repository that is published with GitHub Pages, but your fork is not yet being published.</p>
<p>If that’s right, it’s most likely becuase GitHub Pages has yet to be enabled on your fork. GitHub Pages is not enabled by default on new forks, even if it was enabled on the fork’s parent.</p>
<p>To enable GitHub Pages, you’ll need to set a publishing source from your fork’s settings tab. This help article explains the process in more detail:</p>
<p><a href="https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/" rel="nofollow noopener">https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/</a></p>
<p>Once that’s done, your site should be published at <a href="https://primetime5502.github.io/CityGuesser" rel="nofollow noopener">https://primetime5502.github.io/CityGuesser</a> within about 10 minutes. If it still isn’t showing up after that time, you might need to clear your browser’s cache.</p>
<p>I hope that helps! Good luck for your Hackathon!</p>
